cli/create.go

```diff
@@ -60,7 +60,11 @@ func (r *RootCmd) create() *serpent.Command {
 				workspaceName, err = cliui.Prompt(inv, cliui.PromptOptions{
 					Text: "Specify a name for your workspace:",
 					Validate: func(workspaceName string) error {
-						_, err = client.WorkspaceByOwnerAndName(inv.Context(), codersdk.Me, workspaceName, codersdk.WorkspaceOptions{})
+						err = codersdk.NameValid(workspaceName)
+						if err != nil {
+							return xerrors.Errorf("Workspace name %s", err)
```

Comment:
Every where you do these errors you should do something like: `"workspace name %q is invalid: %w"`

Error wraps should also always be used with `%w` and be at the end with a colon in front, like `"message: %w"`, and error messages should not start with a capital letter